src/aequitas/flow/methods/preprocessing/label_flipping.py

```diff
@@ -203,9 +212,13 @@ def _score_instances(self, X: pd.DataFrame, y: pd.Series) -> pd.Series:
         return scores
 
     def _calculate_prevalence_disparity(self, y: pd.Series, s: pd.Series):
-        prevalence = y.mean()
-        group_prevalence = y.groupby(s).mean().to_dict()
-        group_disparity = {k: v - prevalence for k, v in group_prevalence.items()}
+        prevalences = y.groupby(s).mean()
+        max_prevalence = max(prevalences)
```

Comment:
Instead of doing the disparity relative to the max disparity, why not for the mean? :thinking: